.github/workflows/ci.yml

```diff
@@ -778,8 +780,8 @@ jobs:
     needs: [build-info, wait-for-ci-images]
     strategy:
       matrix:
-        python-version: "${{fromJson(needs.build-info.outputs.python-versions)}}"
-        postgres-version: "${{fromJson(needs.build-info.outputs.postgres-versions)}}"
+        python-version: ["3.7"]
```

Comment:
This is a bit dangerous and not needed. For most cases (as of last week) the exclude matrices are only selecting "representative" combos for tests - so you do not save by hard-conding those I think.

Reply:
yeah both this and putting `test-postgres` everywhere was *just* for testing